src/ant/generic-contract/GenericContract.tsx

```diff
@@ -6,13 +6,12 @@ import { BaseContract, ContractFunction } from 'ethers';
 import { FunctionFragment } from 'ethers/lib/utils';
 import React, { FC, PropsWithChildren, ReactElement, useEffect, useState } from 'react';
 
-import { DisplayVariable } from './DisplayVariable';
-import { FunctionForm } from './FunctionFrom';
-import { NoContractDisplay } from './NoContractDisplay';
+import { DisplayVariable, FunctionForm, NoContractDisplay } from '~~/ant/generic-contract';
```

Comment:
@joaobrunoah please use the local relative import as you might have recursive issues importing from the local index file in the folder

Reply:
Done!